docs/source/CLI_Developer_Guide/Preparing_Your_Data_and_Hyper_Parameters.md

```diff
@@ -10,4 +10,4 @@ The way you construct your code may differ depending on the type of algorithm or
 
 - Example applications 
 
-For example, see helpful [documentation on TetraOpt](https://terra-quantum-tq42sdk-docs.readthedocs-hosted.com/en/latest/TetraOpt_Algorithm/Introduction.html#) - a global optimization library based on tensor train (TT) decomposition.
\ No newline at end of file
+For example, see helpful [documentation on TetraOpt](https://help.terraquantum.io/) - a global optimization library based on tensor train (TT) decomposition.
```

Comment:
you forgot to adapt the text for the link (this is not a documentation on TetraOpt)